### ISSUE TITLE
Is it possible to add support for Girl Stuck in a Hole?

### ISSUE THREAD

**Issue by noob-ZY** [Sauceke/LoveMachine#125]
[Girl Stuck in a Hole](https://www.dlsite.com/maniax/work/=/product_id/RJ317820.html/?locale=en_US)
Maybe not famous, but still a good game.

**Comment by Sauceke** [comment]
Isn't all movement in this game controlled by the player? If so, I'm assuming you want to play it using depth control devices?

**Comment by noob-ZY** [comment]
> 这个游戏的所有动作不都是由玩家控制的吗？如果是这样，我假设您想使用深度控制设备播放它？

Yes, it's mostly player controlled, and there's actually no need to make a plugin for this game. I'm just curious if it's possible for games like this where the action poses are basically controlled by the player to be supported by lovemachine.
Because I have seen a solution about VaM, which can control the toy according to the position of the object in real time, and the control delay seems to be quite low. I don't know whether its implementation is very different from lovemachine, because it is written in the readme of lovemachine that the implementation needs to analyze the loop animation.
[ToySerialController+VAMLaunch 10](https://hub.virtamate.com/resources/toyserialcontroller-vamlaunch.12663/)

**Comment by Sauceke** [comment]
Yes, lovemachine needs either an animator or _something_ that loops around, sooner or later.
OTOH, this could make a great test animal for the depth control stuff in the [Experiments](https://github.com/Sauceke/LoveMachine.Experiments) plugin. A game where the npc reacts to user input in a realistic way, just what I needed. So I'm gonna move this issue to Experiments.
By the way, if you found this plugin useful, don't forget to give it a ⭐.
